src/render/tutorials/caching-http-requests/2_0.html.md

```diff
@@ -0,0 +1,335 @@
+---
+title: Caching HTTP requests
+author: oskarjerzyk
```

Comment:
I think you also have to provide file in `team/contributors`

Reply:
fixed